integration/basic_test.go

```diff
@@ -264,6 +264,29 @@ func (s *RunSuite) TestBuild(c *C) {
 	c.Assert(two.Config.Cmd, DeepEquals, []string{"echo", "two"})
 }
 
+func (s *RunSuite) TestBuildWithNoCache(c *C) {
```

Comment:
I think the integration test(s) should do more than that ; something like :
- A test for cached build :
  - `build` the image (without the `--no-cache`)
  - `build` the image again (without the `--no-cache`) and make sure we see `Using cache` somewhere
- A test for the `no-cache` build :
  - `build` the image (without the `--no-cache`)
  - `build` the image with `--no-cache` and make sure we don't see `Using cache` messages.
 
The actual test that the `build` work and did not fail with the `--no-cache` flag, but does not check at all if it used the cache or not :stuck_out_tongue_closed_eyes:.